change siteState to registration and update hero date

kee5625 opened the pull request:
changed hero.astro and index.astro files.

kee5625 commented:
New tasks accomplished:
![image](https://github.com/user-attachments/assets/ce9dfa2c-77b9-4bcf-a660-b06cbe0aad09)
+ Updated MLH Trust Badge


Changes reflected:
![image](https://github.com/user-attachments/assets/a4529d6a-afce-46bf-93dd-98403e342d64)

![image](https://github.com/user-attachments/assets/fd951a06-60a5-47a2-b79f-d9152e20c2f9)

![image](https://github.com/user-attachments/assets/66cb421d-479e-4b3e-ac44-094d35a4524f)

chaung844 commented on src/components/nav/NavHamburg.astro:
```diff
@@ -17,13 +17,13 @@ const navLinks = {
     id="mlh-trust-badge" 
-    style="display:block;max-width:100px;min-width:60px;position:fixed;left:50px;top:0;width:10%;z-index:10000" 
-    href="https://mlh.io/na?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2025-season&utm_content=blue" 
+    style="display:block;max-width:100px;min-width:60px;position:fixed;right:50px;top:0;width:10%;z-index:10000"
```
Can you move the trust badge to the left using `left:50px` instead of `right:50px`? It's currently overlapping the hamburger nav bar icon on the right. Everything else looks good to me. Thanks!

kee5625 commented:
Added linkedin url field under github in index.astro:
![image](https://github.com/user-attachments/assets/f7b19589-6ea4-43f8-946f-235736a2f762)

added linkedin url field to json file inside api.ts:
![image](https://github.com/user-attachments/assets/492e6737-b324-493e-834c-1181e30a4e49)


result:
![image](https://github.com/user-attachments/assets/0c68e6bc-fc5d-405d-bbbf-0819708c43d8)


im unsure of the naming conventions though.
